src/gui/base/Checkbox.ts

```diff
@@ -41,7 +42,7 @@ export class Checkbox implements Component<CheckboxAttrs> {
 				},
 			},
 			m(
-				"label.break-all",
+				`label${Checkbox.getBreakClass(a.label())}`,
```

Comment:
i'd much rather have this in the class property of the attrs as well instead of the selector

Reply:
Any particular reason why? I can't find any info on the differences between using the `class` property & dynamically updating the tag. There isn't anything in the guidelines about this either. The `class` property already contains a dynamic class so the resulting code will be messy.

Reply:
We've had security problems with building dynamic mithril selectors. That's really only a problem when using the attributes feature with user input, but making the selector that's capable of producing arbitrary html itself static makes it way easier to confirm at a glance that it's impossible to be causing problems. The class attribute is just used as-is without fancy parsing.

I also realize that I'm more careful than the guidelines propose and that we have tons of old code where the selector is still built on demand, so I'm ready to be told that this stance is overly cautious :D

Reply:
Fair enough. I don't think that's unreasonably cautious given it's the same effort to put dynamic classes in the `class` property as putting it in the mithril selector. In that case, I would propose adding this to the coding guidelines. Using the `class` property instead would be a better habit to get into for the security reasons you just described.